doc/src/guide.rst

```diff
@@ -1,593 +0,0 @@
-.. _guide:
-
-==================
-SymPy User's Guide
-==================
-
-.. role:: input(strong)
-
-Introduction
-============
-
-If you are new to SymPy, start with the :ref:`Tutorial <tutorial>`. If you went
-through it, now
-it's time to learn how SymPy works internally, which is what this guide is
-about. Once you grasp the ideas behind SymPy, you will be able to use it
-effectively and also know how to extend it and fix it.
-You may also be just interested in :ref:`SymPy Modules Reference <module-docs>`.
-
-Learning SymPy
-==============
-
-Everyone has different ways of understanding the code written by others.
-
-Ondřej's approach
------------------
-
-Let's say I'd like to understand how ``x + y + x`` works and how it is possible
-that it gets simplified to ``2*x + y``.
-
-I write a simple script, I usually call it ``t.py`` (I don't remember anymore
-why I call it that way)::
-
-    from sympy.abc import x, y
-
-    e = x + y + x
-
-    print(e)
-
-And I try if it works
-
-.. parsed-literal::
-
-    $ :input:`python t.py`
-    y + 2*x
-
-Now I start `winpdb <http://winpdb.org/>`_ on it (if you've never used winpdb
--- it's an excellent multiplatform debugger, works on Linux, Windows and Mac OS
-X):
-
-.. parsed-literal::
-
-    $ :input:`winpdb t.py`
-    y + 2*x
-
-and a winpdb window will popup, I move to the next line using F6:
-
-.. image:: pics/winpdb1.png
```

Comment:
I see there are some associated images for this tutorial. If they are not used anymore they should be removed manually too

Reply:
Done.